loader/full-struct_test.go

```diff
@@ -866,7 +866,7 @@ func fullExampleYAML(workingDir, homeDir string) string {
     - type: tmpfs
       target: /opt
       tmpfs:
-        size: 10000
+        size: "10000"
```

Comment:
Is this change intentional?

Reply:
yes, using UnitBytes makes the marshaller always prefer string